### PR TITLE
Handle the case where the glob matches exactly one file

### PR DESCRIPTION
**Pull request by TAGraves**
I was running into a case with a customer where the glob was matching exactly one file and the upload was failing as a result. It looks like in the case where the glob returned exactly one file, we were assuming that there was no `*` in the path at all. We don't need to make that assumption.

### CODE REVIEW

File: src/run.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default async function run(): Promise<void> {
     const artifacts: Artifact[] = inputs.artifacts
       .flatMap(artifact => {
         const expandedGlob = fastGlob.sync(artifact.path)
-        if (expandedGlob.length > 1) {
+        if (expandedGlob.length > 0) {
```
Comment (michaelglass) [inline src/run.ts:28]: Yup. Good catch

File: src/run.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export default async function run(): Promise<void> {
...
           return expandedGlob.map(path => ({
             ...artifact,
             path,
```
Comment (kylekthompson) [inline src/run.ts:31]: This makes sense to me- in the case where it _doesn't_ find anything in the glob, we'll store the glob-path as the original path and create an artifact with missing files?

**Review by kylekthompson — APPROVED** [review]

File: src/run.ts (hunk 2 above)
Comment (TAGraves) [reply src/run.ts:31]: Yep!

**Comment by michaelglass** [comment]
whoops, thanks for the fix!

**Pull request merged**